oxidation/libparsec/crates/tests_fixtures/src/lib.rs

```diff
@@ -168,7 +168,7 @@ impl std::ops::Deref for TmpPath {
 
 impl Drop for TmpPath {
     fn drop(&mut self) {
-        std::fs::remove_dir_all(&self.0).unwrap();
+        remove_dir_all::remove_dir_all(&self.0).unwrap();
```

Comment:
why `std::fs::remove_dir_all` is not sufficient ?

Reply:
`The current Windows implementation of remove_dir_all in the Rust standard library has a long-standing issue with consistently deleting directories.`
Several crates have been created to overcome this problem